DecipherNow/gm-fabric-dashboard
# 1145 stylelint

Resolves #1145 

## Proposed Changes

  - Add Stylelint and associated tooling
  - Configure to lint styled-components via `npm run lint-css`
  - Enable in-browser highlighting in VSCode via the `stylelint` plugin
  
## How to Test

  - In VS Code, install the `stylelint` extension and restart VSCode
  - `npm install`
  - `npm run lint-css`
 - Open a file flagged with issues in VSCode, you should see issues in the "problems" tab and context squiglies about the errors
- Finally, please run the app and visually confirm that my linting fixes didn't cause UI regressions

Note: There are some flagged CSS syntax errors that I didn't fix because it was unclear how the rules should actually behave. I intend to open follow-up issues on these and assign them to the last person to touch the styled-components.
